e2e-tests/tfm/cypress/e2e/pages/amendments/amendmentsPage.js

```diff
@@ -37,16 +39,57 @@ const amendmentsPage = {
   amendmentFacilityEndDateDayInput: () => cy.get('[data-cy="amendment--facility-end-date-day"]'),
   amendmentFacilityEndDateMonthInput: () => cy.get('[data-cy="amendment--facility-end-date-month"]'),
   amendmentFacilityEndDateYearInput: () => cy.get('[data-cy="amendment--facility-end-date-year"]'),
+  amendmentCurrentFacilityEndDate: () => cy.get('[data-cy="amendment--current-facility-end-date"]'),
+  amendmentFacilityEndDateDetails: () => cy.get('[data-cy="amendment--facility-end-date-details"]'),
+
+  amendmentBankReviewDateDayInput: () => cy.get('[data-cy="amendment--bank-review-date-day"]'),
+  amendmentBankReviewDateMonthInput: () => cy.get('[data-cy="amendment--bank-review-date-month"]'),
+  amendmentBankReviewDateYearInput: () => cy.get('[data-cy="amendment--bank-review-date-year"]'),
+  amendmentCurrentBankReviewDate: () => cy.get('[data-cy="amendment--current-bank-review-date"]'),
+  amendmentBankReviewDateDetails: () => cy.get('[data-cy="amendment--bank-review-date-details"]'),
+
+  navigateToIsUsingFacilityEndDatePage: ({ startNewAmendment = false, changeFacilityValue = false, newCoverEndDate = today } = {}) => {
```

Comment:
I'm having a think about this --
It's only applicable for this page, so this should be only in a pages file? Otherwise it's easy to invoke it when not on the amendments page?

Unless you mean this should be a command nested inside the `amendmentsPage.js` file? In which case, we would still have the issue of it being invoked when not actually allowed?

Keen to hear thoughts

Reply:
Raised in slack as per other comments on similar comments